sktime/forecasting/compose/tests/test_reduce_global.py

```diff
@@ -358,3 +358,30 @@ def test_nofreq_pass():
     np.testing.assert_almost_equal(
         y_pred_global["c0"].values, y_pred_nofreq["c0"].values
     )
+
+
+def test_timezoneaware_index():
+    y = load_solar(api_version=None)
+    y_notz = y.copy().tz_localize(None)
```

Comment:
Non-blocking suggestions: `assert y.index.tz is not None` and `assert y_notz.index.tz is None`.

Reply:
added 👍